projects/plugins/jetpack/modules/wordads/class-wordads.php

```diff
@@ -435,13 +435,20 @@ public function insert_head_meta() {
 		// Remove linebreaks and sanitize.
 		$tag_top = esc_js( str_replace( array( "\n", "\t", "\r" ), '', $tag_top ) );
 
+		// Get sidebar tag.
+		$tag_sidebar = $this->get_fallback_ad_snippet( $section_id, 'skyscraper', 'sidebar', '', '{{unique_id}}' );
+
+		// Remove linebreaks and sanitize.
+		$tag_sidebar = esc_js( str_replace( array( "\n", "\t", "\r" ), '', $tag_sidebar ) );
+
 		// phpcs:disable WordPress.Security.EscapeOutput.HeredocOutputNotEscaped
 		echo <<<HTML
 				<script type="text/javascript">
 					var sas_fallback = sas_fallback || [];
 					sas_fallback.push(
 						{ tag: "$tag_inline", type: 'inline' },
 						{ tag: "$tag_belowpost", type: 'belowpost' },
+						{ tag: "$tag_sidebar", type: 'sidebar' },
```

Comment:
I'm adding it to support the legacy sidebar widget.

Reply:
_Per our conversation, I think we can add a TODO for this instead, as we haven't migrated/tested the legacy sidebar in this setup yet_